### ISSUE TITLE
Registration link for the MIDL mailing list

### ISSUE THREAD

**Issue by HKervadec** [MIDL-Conference/midl-website-2021#3]
As we figuring out in parallel how to hand-over the mailing list, I think it would be nice to have a registration link on the website, for both:
* MIDL 2021 updates
* MIDL society updates

Right now there is about 2500 people in the list, coming only from the MIDL 2020 registrations. 

This was one of the major communication issue of this year, we focused a too much on Twitter and didn't use other channels enough. The in-house mailing list should help a lot.
